### ISSUE TITLE
(Release 3.3) broken link in new lineage lab

### ISSUE THREAD

**Issue by planetf1** [odpi/egeria#5864]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Link to exposing a service in

Important: When running this lab using kubernetes deployment, make sure that you expose the Egeria UI running in the container to your local network and access it via localhost.

Is broken - it points to https://odpi.github.io/egeria-docs/guides/admin/kubernetes/chart_lab/#accessing-the-egeria-ui which no longer exists (I think some refactoring - not me - was done which changed the URLs)

We could fix in 3.3 if quick, or add to release notes
cc: @lap

### Expected Behavior

Link should work correctly

### Steps To Reproduce

as above

### Environment

```markdown
- Egeria:
- OS:
- Java:
- Browser (for UI issues):
- Additional connectors and integration:
```

**Comment by lpalashevski** [comment]
@planetf1 I will look into it

**Comment by planetf1** [comment]
Additionally, in ui-asset-search notebook
 a) We also need the link to how to expose the kubernetes service
 b) The password provided (admin) is wrong - should be secret

**Comment by lpalashevski** [comment]
> Additionally, in ui-asset-search notebook a) We also need the link to how to expose the kubernetes service (the process provided does not work, as it provides the cluster service address, which isn't necessarily exposed...) b) The password provided (admin) is wrong - should be secret
> 
> Can handle in release notes if needed

I included quick-fix for these two in the same PR #5865
